### DISCUSSION TITLE
Compare Best Laps

### DISCUSSION THREAD

**Discussion by Maximo1970** [snipem/gt7dashboard#12] [Ideas]
Firstly, thanks for taking the time to create this. This adds another dimension to GT7, keep up the great work.

Like most of us, I race with a number of friends. It would be great if we could directly compare out best laps on the same page. Very much like you have at the moment with Reference Lap and Last lap.

Thanks,

Garry

**Comment by snipem** [comment]
You can select multiple laps by clicking them in the bottom lap list

**Reply by Maximo1970** [reply]
Thanks for the reply, although I don't think you've understood my comment.

What I would to do is compare my best lap against a friend's best lap, on the same screen, much like you can compare your own individual laps from a session. Is there a way to merge data from two separate sources?

**Reply by snipem** [reply]
This is not possible at the moment. Laps are saved in a binary format (Python pickles). If they could be merged by a third party tool, you could merge your own laps with the laps of others and display them in gt7dashboard.

I am thinking about using JSON-based human readable lap files instead of the binary ones. They could be merged easily. Either by hand or tool assisted. I will gladly accept any merge request on this.
